modin/pandas/test/test_io.py

```diff
@@ -524,6 +567,61 @@ def test_read_csv_delimiters(
             **kwargs,
         )
 
+    # Column and Index Locations and Names tests
+    @pytest.mark.xfail(
+        Engine.get() != "Python",
+        reason="many parameters combiantions fails: issue #2312, #2307",
+    )
+    @pytest.mark.parametrize("header", ["infer", None, 0])
+    @pytest.mark.parametrize("index_col", [None, "col1"])
+    @pytest.mark.parametrize("prefix", [None, "_", "col"])
+    @pytest.mark.parametrize(
+        "names", [None, ["col1"], ["c1", "c2", "c3", "c4", "c5", "c6", "c7"]]
+    )
+    @pytest.mark.parametrize(
+        "usecols", [None, ["col1"], ["col1", "col2", "col6"], [0, 1, 5]]
+    )
+    @pytest.mark.parametrize("skip_blank_lines", [True, False])
+    def test_read_csv_col_handling(
+        self,
+        request,
+        header,
+        index_col,
+        prefix,
+        names,
+        usecols,
+        skip_blank_lines,
+    ):
+        if request.config.getoption("--simulate-cloud").lower() != "off":
+            pytest.xfail("The reason of tests fail in `cloud` mode is unknown for now")
+        kwargs = {
+            "header": header,
+            "index_col": index_col,
+            "prefix": prefix,
+            "names": names,
+            "usecols": usecols,
+            "skip_blank_lines": skip_blank_lines,
+        }
+
+        eval_io(
+            filepath_or_buffer=pytest.csvs_names["test_read_csv_blank_lines"],
+            fn_name="read_csv",
+            **kwargs,
+        )
+
+    @pytest.mark.xfail(reason="infinite recursion error - issue #2032")
+    def test_read_csv_squeeze(self, make_csv_file):
```

Comment:
Am I right, that we're not using `make_csv_file` fixture inside `test_read_csv_squeeze`?

Reply:
Yes, fixture was removed from `test_read_csv_squeeze`.